config/nrfconnect/chip-module/Kconfig

```diff
@@ -26,4 +26,8 @@ config CHIP_NFC_COMMISSIONING
 	imply NFC_NDEF_URI_REC
 	imply NFC_NDEF_URI_MSG
 	help
-	  Enables NFC commissioning by sharing onboarding payload in NFC tag.
\ No newline at end of file
+	  Enables NFC commissioning by sharing onboarding payload in NFC tag.
+
+config CHIP_DEVICE_AVERAGE_CURRENT_CONSUMPTION
```

Comment:
It's fine for TE7, but we should put in in Kconfig of the matter_weather_station rather than the entire CHIP.

Reply:
Ok, I will keep it in mind when moving to the NCS